SportHub.API/Controllers/AuthController.cs

```diff
@@ -0,0 +1,97 @@
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using SportHub.Business;
+using SportHub.Data;
+using SportHub.Business;
+
+namespace SportHub.Controllers;
+
+[AllowAnonymous]
+[ApiController]
+[Route("[controller]")]
+public class AuthController : ControllerBase
+{
+    private readonly IUserService _userService;
+    private readonly IJwtService _jwtService;
+    
+    public AuthController(IUserService userService, IJwtService jwtService)
+    {
+        _userService = userService;
+        _jwtService = jwtService;
+    }
+
+    [HttpPost("login")]
+    [AllowAnonymous]
+    public IActionResult LogIn(string email, string password)
+    {
+        var user = _userService.GetUserByEmailAsync(email).Result;
+
+        if (user == null || user.Password != password)
+        {
+            return Unauthorized();
+        }
+
+        JwtResponse response = _jwtService.GenerateTokens(user).Result;
+        
+        Response.Cookies.Append(
+            "refreshToken",
+            response.RefreshToken,
+            new CookieOptions
+            {
+                HttpOnly = true,
+                Secure = true,
+                SameSite = SameSiteMode.Strict
+            }
+        );
+
+        return Ok(response);
+    }
+    
+    [HttpPost("refresh")]
+    public async Task<IActionResult> Refresh()
+    {
+        var refreshToken = Request.Cookies["refreshToken"];
```

Comment:
Bit complecated -> you can move into one method in JWTService (You can reuse UserService  inside JWTService)

Reply:
I don't understand what you mean

Reply:
We need one thing - response with token, in other cases we return Unauthorized